frontend/src/employee-frontend/components/vasu/templates/VasuTemplatesPage.tsx

```diff
@@ -82,6 +82,7 @@ export default React.memo(function VasuTemplatesPage() {
                         <IconButton
                           icon={faCopy}
                           onClick={() => setTemplateToCopy(template)}
+                          disabled={true} /*needs fixing*/
```

Comment:
What does this mean?

Reply:
the template copying doesnt make sense anymore after some changes, going to discuss with Kimi should we remove it or how to fix. I can remove this button for now though instead of disabling